vcbuild.bat

```diff
@@ -437,7 +445,7 @@ echo Failed to create vc project files.
 goto exit
 
 :help
-echo vcbuild.bat [debug/release] [msi] [test-all/test-uv/test-inspector/test-internet/test-pummel/test-simple/test-message] [clean] [noprojgen] [small-icu/full-icu/without-intl] [nobuild] [sign] [x86/x64] [vc2015] [download-all] [enable-vtune] [lint/lint-ci]
+echo vcbuild.bat [debug/release] [msi] [test-all/test-uv/test-inspector/test-internet/test-pummel/test-simple/test-message] [clean] [noprojgen] [small-icu/full-icu/without-intl] [nobuild] [sign] [x86/x64] [vc2015] [download-all] [enable-vtune] [lint/lint-ci/jslint/jslint-ci]
```

Comment:
Good point, I didn't notice that there isn't an option to just run `cpplint` (although there is a `cpplint` variable. Maybe there should be, there is in [the Makefile](https://github.com/nodejs/node/blob/master/Makefile#L883) and it makes sense to have them mirror each other if possible. Doesn't have to be done in this PR though!